codemeta.json

```diff
@@ -6,7 +6,7 @@
     "contIntegration": "https://github.com/NeuralEnsemble/elephant/actions",
     "dateCreated": "2013-17-15",
```

Comment:
There's also something wrong with this date

Reply:
Yes indeed, thanks for pointing this out.